.github/workflows/ci.yml

```
@@ -0,0 +1,292 @@
+  backend:
+    name: Backend Tests
+    runs-on: ubuntu-latest
+    
+    steps:
+    # Step 1: Get the code from repository
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    # Step 2: Install Rust toolchain for compilation
+    - name: Install Rust toolchain
+      uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)